### PR TITLE
Ros2 v0.8 path_test_publisher

### PR DESCRIPTION
**Pull request by isamu-takagi**

### CODE REVIEW

File: planning/path_test_publisher/package.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
```
Comment (wep21) [inline planning/path_test_publisher/package.xml:2]: It's better to use format 3.

**Comment by wep21** [comment]
test result
``` 
ros2 topic echo /path
header:
  stamp:
    sec: 1613468320
    nanosec: 724778518
  frame_id: map
points:
- pose:
    position:
      x: 0.0
      y: 0.0
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.0
      w: 1.0
  twist:
    linear:
      x: 10.0
      y: 0.0
      z: 0.0
    angular:
      x: 0.0
      y: 0.0
      z: 0.0
  type: 0
```

File: planning/path_test_publisher/package.xml (hunk 1 above)
Comment (isamu-takagi) [reply planning/path_test_publisher/package.xml]: Fixed.

**Review by wep21 — APPROVED** [review]
LGTM

**Pull request merged**